test/test_sort_and_select.py

```diff
@@ -175,6 +175,14 @@ def test_sort_stable_none(self):
         y = x.sort(stable=None).values
         self.assertTrue(torch.all(y == torch.ones(10)).item())
 
+    @onlyCPU
```

Comment:
Why onlycpu? Sorting complex values is semantically meaningless on GPUs as well?

Reply:
The GPU codepath already errors out, and has its own error.

Reply:
```
  TORCH_CHECK(self.sizes()[dim] <= std::numeric_limits<int>::max(),
    "The dimension being sorted can not have more than INT_MAX elements.");

  const auto self_dtype = self.dtype();
  TORCH_CHECK(self_dtype != ScalarType::ComplexFloat && self_dtype != ScalarType::ComplexDouble,
    "Sort currently does not support complex dtypes on CUDA.");

```

I guess I could just hoist this code to a generic codepath and enable for GPU. Let me see what I can do.